conans/test/functional/scm/scm_test.py

```diff
@@ -899,16 +901,6 @@ def build(self):
         self.assertIn("SOURCE METHOD CALLED", self.client.out)
         self.assertIn("BUILD METHOD CALLED", self.client.out)
 
-    def test_scm_serialization(self):
-        data = {"url": "myurl", "revision": "23", "username": "myusername",
-                "password": "mypassword", "type": "svn", "verify_ssl": False,
-                "subfolder": "mysubfolder"}
-        conanfile = namedtuple("ConanfileMock", "scm")(data)
-        scm_data = SCMData(conanfile)
-        the_json = str(scm_data)
-        data2 = json.loads(the_json)
-        self.assertEqual(data, data2)
```

Comment:
This test was duplicated (see `scm_serialization_test` <-- it is the one modified above in this file)